src/transactions/feed/TransactionFeedV2.tsx

```diff
@@ -499,13 +505,11 @@ export default function TransactionFeedV2() {
         scrollEventThrottle={16}
         refreshControl={
           <RefreshControl
-            refreshing={isRefreshingBalances}
-            onRefresh={onRefresh}
+            refreshing={status === 'loading'}
```

Comment:
@kathaypacific we already have `isFetching` that signalizes that the feed is refreshing. What do you think of renaming the `loading` status to `loadingBalances` and modifying the check to `status === 'loadingBalances' && isFetching`? It might be a bit more verbose but it will make it more clear why we don't just use `isFetching` as a source of truth for this check.

Reply:
@kathaypacific gotcha! I haven't thought about the constant layout shift, this makes sense now!